SteamList/Controllers/GameDetailsViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  GameDetailsViewController.swift
+//  SteamList
+//
+//  Created by Macbook on 24/01/2023.
+//
+
+import UIKit
+
+class GameDetailsViewController: UIViewController {
+    
+    private let scrollView: UIScrollView = {
+        var scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return scrollView
+    }()
+    
+    private let gameInfoViewController: GameInfoViewController = {
+        var viewController = GameInfoViewController()
+        viewController.view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return viewController
+    }()
+    
+    private let gameDescriptionViewController: GameDescriptionViewController = {
+        var viewController = GameDescriptionViewController()
+        viewController.view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return viewController
+    }()
+    
+    
+    var constraints = [NSLayoutConstraint]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        add(gameInfoViewController)
+        add(gameDescriptionViewController)
+        constructHirarchy()
+        self.title = "Games"
+    }
+    
+    override func viewDidLayoutSubviews() {
```

Comment:
You can add constraints without waiting this method gets called

Reply:
@AbboskhonShukurullaev advised me to put constraints into this function